src/Normalizer/NormalizerBase.php

```diff
@@ -36,8 +22,8 @@ public function supportsDenormalization($data, $type, $format = NULL) {
   /**
    * {@inheritdoc}
    */
-  public function denormalize($data, $class, $format = NULL, array $context = array()) {
-    return FALSE;
+  public function denormalize($data, $class, $format = NULL, array $context = []) {
+    throw new \RuntimeException('Denormalization is not supported.');
```

Comment:
\Drupal\serialization\Normalizer\NormalizerBase returns a boolean for this. Is the idea here that:

1. This will get a 405 response to HTTP requests before it gets to this exception.
2. Developers that attempt to denormalize a schema should be told they are doing something wrong.